torch/nn/init.py

```diff
@@ -0,0 +1,240 @@
+import numpy as np
```

Comment:
numpy isn't currently a dependency of PyTorch. It looks like you can substitute `math` and tensor operations for the usages.